planner/core/explain.go

```diff
@@ -429,6 +431,7 @@ func (p *PhysicalIndexJoin) explainInfo(normalized bool, isIndexMergeJoin bool)
 		for i := range p.OuterHashKeys {
 			expr, err := expression.NewFunctionBase(MockContext(), ast.EQ, types.NewFieldType(mysql.TypeLonglong), p.OuterHashKeys[i], p.InnerHashKeys[i])
 			if err != nil {
+				log.Warn("NewFunctionBase", zap.Error(err))
```

Comment:
Why we are not using `logutil.BgLogger` like other places?

Reply:
I think we can add some information to help to locate this if it appears in the log, like the method name "(*PhysicalIndexJoin).explainInfo".

Reply:
> Why we are not using `logutil.BgLogger` like other places?

Ok, I have updated.